Dockerfile

```diff
@@ -0,0 +1,39 @@
+ARG RUBY_IMAGE
+FROM ${RUBY_IMAGE}
+
+ARG USER_ID_GROUP
+ARG FRAMEWORKS
+ARG VENDOR_PATH
+ARG BUNDLER_VERSION
+
+# For tzdata
+# ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update -qq \
+      && apt-get install -qq -y --no-install-recommends \
+        build-essential libpq-dev git \
+      && rm -rf /var/lib/apt/lists/*
+
+# Configure bundler and PATH
+ENV LANG=C.UTF-8
+
+ENV GEM_HOME=$VENDOR_PATH
+ENV BUNDLE_PATH=$GEM_HOME \
+  BUNDLE_JOBS=4 BUNDLE_RETRY=3
+ENV BUNDLE_APP_CONFIG=$BUNDLE_PATH \
+  BUNDLE_BIN=$BUNDLE_PATH/bin
+ENV PATH=/app/bin:$BUNDLE_BIN:$PATH
+
+ENV FRAMEWORKS $FRAMEWORKS
+
+RUN mkdir -p $VENDOR_PATH && \
```

Comment:
Cosmetic feedback:

This could be potentially moved to the very end and use the USER directive rather than using the chown 
command?

```Dockerfile
...
WORKDIR /app

USER $USER_ID_GROUP
RUN mkdir -p $VENDOR_PATH
```

Although I'm not sure whether the value for the USER_ID_GROUP varible might be based on the `gid:usid` format.

Reply:
I couldn't make `USER` work, so reverted to the `chown` approach